samples/excel/85-preview-apis/range-direct-dependents.yaml

```
@@ -0,0 +1,186 @@
+order: 4
+id: excel-direct-dependents
+name: Direct dependents
+description: This sample shows how to find and highlight the direct dependents of the currently selected cell. Dependents are cells affected by the value in a cell.
+host: EXCEL
+api_set:
+    ExcelAPI: '1.13'
+script:
+  content: |
+    $("#setup").click(() => tryCatch(setup));
+    $("#select-first-cell").click(() => tryCatch(selectFirstCell));
+    $("#select-second-cell").click(() => tryCatch(selectSecondCell));
+    $("#get-direct-dependents").click(() => tryCatch(getDirectDependents));
+    $("#clear-highlighting").click(() => tryCatch(clearFormatting));
+    
+    /** Select a cell with direct dependents on the same worksheet. */
+    async function selectFirstCell() {
+      await Excel.run(async (context) => {
+        // Ensure correct worksheet is active.
```

Consider making this more direct (if you take this suggestion, please apply it to the other selection method).
```suggestion
        // Activate the sample worksheet.
```